### PR TITLE
Disk free space

### PR DESCRIPTION
**Pull request by guiniol**
This PR adds a module to show the remaining free space on a given disk. The disk is chosen by filesystem path.

### CODE REVIEW

**Comment by pydsigner** [comment]
How about using https://docs.python.org/3.3/library/os.html#os.statvfs to avoid the library dependency?

**Comment by guiniol** [comment]
I'll look into it. I do want to remove the psutil dependency where possible.

File: py3status/modules/disk_free_space.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    format: format string
+        (default: "{space:.1f}G")
+
+Format string parameters:
+    {space} space left (in GB)
+    {path} path checked
+"""
+
+# import your useful libs here
+import psutil
```
Comment (tobes) [inline py3status/modules/disk_free_space.py]: Is there a reason you are using psutil?  I've got a disk space module that uses `df` to get the info.

Comment (tobes) [inline py3status/modules/disk_free_space.py]: opps just saw @pydsigner's comment

**Comment by guiniol** [comment]
Closing this until #476 gets merged.

**Pull request closed without merging**